src/Icons/tests/Unit/Registry/InMemoryIconRegistryTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Icons\Tests\Unit\Registry;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\UX\Icons\Registry\InMemoryIconRegistry;
+use Symfony\UX\Icons\Svg\Icon;
+
+/**
+ * @author Simon André <smn.andre@gmail.com>
+ */
+final class InMemoryIconRegistryTest extends TestCase
```

Comment:
`throw new IconNotFoundException(sprintf('Icon "%s" not found.', $name));`
is not covered :)